skin.json

```diff
@@ -33,6 +44,19 @@
     "snapwikiskin": ["i18n"]
   },
   "ResourceModules": {
+    "skins.snapwikiskin.search": {
+      "dependencies": ["mediawiki.Uri", "wvui"],
```

Comment:
I think ``vue`` is [needed](https://www.mediawiki.org/wiki/Vue.js#How_to_use_on_the_fly_compilation) here.

Reply:
It works, so I'm assuming wvui or something deals with it.

Reply:
Confirmed. ``wvui``'s module [definition](https://github.com/wikimedia/mediawiki/blob/master/resources/Resources.php#L608) has ``vue`` as a [dependency](https://github.com/wikimedia/mediawiki/blob/master/resources/Resources.php#L616),